### DISCUSSION TITLE
Todays meeting

### DISCUSSION THREAD

**Discussion by ap1987-ap** [munoztd0/DAP_2021#11] [General]
Hi David,

sadly I cant make it today in person for the meeting.
Is it possible to join it via zoom when you meet with the others? Or should we rescedule for me?

Thanks and best,
Anja

**Comment by munoztd0** [comment]
Hello Anja,

Well, if someone else is planning on not coming (@lucilefavero , @danielsgranja , @faureunige ?) we might be able to arrange another meeting but if not, i will send you a zoom link.

**Reply by danielsgranja** [reply]
Well, both work for me, but with the problem i have, I would prefer to come with my computer

**Reply by munoztd0** [reply]
Yeah, if you can come live I suggest you do because that's much easier for both of us.

**Reply by lucilefavero** [reply]
Both work for me too.

**Comment by ap1987-ap** [comment]
Ok, thanks

**Comment by munoztd0** [comment]
@faureunige We will have a meeting on Thursday the 16th at 3pm via [zoom](
 https://unige.zoom.us/j/69919724393) .
Get well!

**Comment by faureunige** [comment]
That’s perfect, thank you so much !

Juliette

De : David Muñoz Tord ***@***.***>
Répondre à : munoztd0/DAP_2021 ***@***.***>
Date : lundi 13 décembre 2021 à 17:46
À : munoztd0/DAP_2021 ***@***.***>
Cc : Juji Faure ***@***.***>, Mention ***@***.***>
Objet : Re: [munoztd0/DAP_2021] Todays meeting (Discussion #11)


@faureunige<https://github.com/faureunige> We will have a meeting on Thursday the 16th at 3pm via zoom<https://unige.zoom.us/j/69919724393> .
Get well!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/munoztd0/DAP_2021/discussions/11#discussioncomment-1799753>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATWBGMVSNE3ZKUXXRBT6G2DUQYPM5ANCNFSM5J53N5EQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
